wordpress-mobile/gutenberg-mobile
# Split the bundle creation steps for release and code update.

Change the bundle step to only create updated bundles and not get
updates strings from the web.
This way for code updates only we just need to run `yarn bundle`.
For the releases, we run `yarn release:bundle`.
This will accelerate the builds for WP-Android and for WP-iOS it will
allow deploy and test of PR that need bundle updates quicker.

To test:
 - Run `yarn bundle` and check that only the bundles are updated with local string
 - Run `yarn release:bundle` check that bundles are updated and we also get new strings that come from the web repo.

PR submission checklist:
